strapi/strapi
# feat(cm): add publish and save

### What does it do?

* adds publish and save ability to CM

### Why is it needed?

* Quick fire publish, zoom zoom.

### How to test it?

* As a user I should be able to create an entity and publish it immediately saving at the same time.

### Related issue(s)/PR(s)

* resolves CONTENT-2202
